.gitignore

```diff
@@ -0,0 +1,5 @@
+node_modules/
+*.log
+dist/
+tests/support/var/
```

Comment:
nit: Let's leave out this line until we've written some tests; we may not end up creating this dir